tasks/Investment Game (hidden profile)

```diff
@@ -0,0 +1,43 @@
+ 
+ > Identified GDSS Condition: When participants in the identified conditions arrived for the experiment, they were introduced to the other members of their group and encouraged to get acquainted while the experimenter finished getting things set up. Group members were seated together in the lab at adjacent work stations, where name placards were placed on top of each participant's monitor. The participants in this condition were instructed that because their electronic communication would be anonymous by default, they would have to add their names to the beginning of each of their comments as they typed. 
+ 
+2. The GDSS groups used a brain-writing tool, which splits each participant's monitor screen into windows for input and display. The entire set of comments entered remained visible at all times, and participants could easily move between scrolling through the comments andentering their own comments. They were given ten minutes to practice before recieving the task instructions 
+
+3. Anonymous Condition: These groups were told explicitly that they would be commuincating anonymously with some of the people in the room and that the other members of their group were scattered in a random pattern. These participants were instructed that their electronic communication would be anonymous by default and that they should not include their names on any of their comments. They could only begin communication when the session began 
+
+4. Paper copies of the task materials and task instructions were distributed to them individually. Within each group, the person to receive the minority information set was selected randomly at the time of assigning the groups to conditions. The remaining three sets of information were distributed in the order of group member seating. 
+
+5. The experimenter then read aloud the task instructions, which the participants followed along on their paper copies. The participants then read silently their copies of the company profiles and provided their private opinions of the correct rank ordering of the three companies on special forms provided. To reinforce the privacy of these decisions, participants were not asked to put theirnames on the forms.
+6. After the individuals recorded their private opinions, groups were then instructed to discuss the three companies and to reach a unanimous decision on a rank order among them. 
+
+7. Groups in the face-to-face conditions then moved immediately to the discussion. The experimenter remained in the room throughout the session, andaudiotaped the discussions, which were transcribed for later analysis. Groups in the GDSS conditions received the brief software training and then moved to the discussion. The groups were not given a definite time limit for their discussion at the outset but were told that most groups took approximately 30 min. The experimenter gave the groups a 10-min warning andstopped the discussion at the end of 45 min, even if a decisionhad not been reached.
+
```

Comment:
Participants in the face-to-face conditions recorded the final group decision on a special form provided, and groups in the GDSS conditions typed the group decision into their computer transcripts. Participants in all conditions provided their current private opinions on special forms provided.